<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<launch>

<!-- move_action package navigates robot without Rviz and plan the robot path with LTL planner -->

	<!-- Use turtlebot as simulation, load gazebo and amcl first -->
	<!-- It has a bit covariance in initialization, the robot needs time to confirm its initial pose -->

	<include file="$(find turtlebot_gazebo)/launch/turtlebot_world.launch">
		<arg name="world_file"  default="$(find move_action)/map/world1.sdf"/>
	</include>

	<include file="$(find turtlebot_gazebo)/launch/amcl_demo.launch">
		<arg name="map_file"  default="$(find move_action)/map/map.yaml"/>
	</include>

    <!--Run nodes-->

	<node name="task_publisher" pkg="move_action" type="task_publisher.py" respawn="true"/>
	<node name="next_move_planner_server" pkg="move_action" type="plan_service.py" respawn="true"/>
	<node name="move_base_action_server" pkg="move_action" type="move_action_service.py" respawn="true"/>

</launch>
